[v1.0.1] Cannot delete/start openpai service correctly

<!-- Please use this template while reporting an issue and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->


**Organization Name**:
Advantech

<!--This information is optional, but you are highly encourage to leave this reference info for us to know better about the context.!-->

**Short summary about the issue/question**:
Cannot delete openpai service correctly

**Brief what process you are following**:
1. Login devbox container
2. Execute ./paictl.py service delete
3. The delete process stop upon the log
```
2021-05-31 03:02:43,793 [INFO] - deployment.paiLibrary.paiService.service_template_generate : Save the generated file to /root/pai/deployment/paiLibrary/paiService/../../../src/frameworkcontroller/deploy/frameworkcontroller-config.yaml.
2021-05-31 03:02:43,831 [INFO] - root : It is not a service deploy file! Only support ['DaemonSet', 'Deployment', 'StatefulSet', 'Pod']
2021-05-31 03:02:43,832 [INFO] - deployment.paiLibrary.paiService.service_template_generate : The template file of service frameworkcontroller is generated.
2021-05-31 03:02:43,832 [INFO] - deployment.paiLibrary.paiService.service_management_delete : Begin to delete service: [ frameworkcontroller ]
2021-05-31 03:02:43,832 [INFO] - deployment.paiLibrary.paiService.service_management_delete : Begin to execute service frameworkcontroller's delete script.
error: the server doesn't have a resource type "frameworks"
Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "frameworks.frameworkcontroller.microsoft.com" not found
2021-05-31 03:02:44,736 [ERROR] - deployment.paiLibrary.common.linux_shell : Failed to execute the delete script of service frameworkcontroller
```

<!--deployment related issues
Please fill this for deployment related issues:
- Operating type: Initial deployment / upgrading / operating etc.
- Brief what deployment process you are following -->

<!--User job related issues
GitHub is not the right place for support requests.
If you're looking for help, check [Stack Overflow](https://stackoverflow.com/questions/tagged/openpai) and the [troubleshooting guide](https://github.com/Microsoft/pai/blob/master/docs/user/troubleshooting_job.md and https://github.com/Microsoft/pai/blob/master/docs/job_tutorial.md#how-to-debug-a-job).
-->

**How to reproduce it**:
Execute ./paictl.py service delete


<!--Fill the following information if your issue need diagnostic support from the team, as minimally and precisely as possible!-->

**OpenPAI Environment**:
- OpenPAI version:
v1.0.1
- OS (e.g. from /etc/os-release):
Ubuntu 16.04

--- a/src/frameworkcontroller/deploy/delete.sh
+++ b/src/frameworkcontroller/deploy/delete.sh
@@ -24,7 +24,10 @@ until [[ $(kubectl get frameworks | wc -l) -eq 0 ]]; do
     kubectl delete --all frameworks
     sleep 5
 done
-kubectl delete customresourcedefinitions frameworks.frameworkcontroller.microsoft.com || exit $?
+
+if kubectl get crd | grep -q 'frameworks.frameworkcontroller.microsoft.com'; then
+    kubectl delete customresourcedefinitions frameworks.frameworkcontroller.microsoft.com || exit $?
+fi
 
 /bin/bash stop.sh || exit $?
 

